tokens/functional/shadow/dark.json5

```diff
@@ -0,0 +1,93 @@
+{
+  shadow: {
+    color: {
+      small: {
+        $value: '{base.color.scale.black}',
+        alpha: 0.96,
+        $type: 'color'
+      },
+      medium: {
+        $value: '{base.color.scale.gray.4}',
+        alpha: 0.85,
+        $type: 'color'
+      },
+      large: {
+        $value: '{base.color.scale.gray.4}',
+        alpha: 0.8,
+        $type: 'color'
+      },
+      extraLarge: {
+        $value: '{base.color.scale.gray.4}',
+        alpha: 0.7,
+        $type: 'color'
+      },
+      highlight: {
+        $value: '{base.color.scale.white}',
+        alpha: 0.25,
+        $type: 'color'
+      },
+      inset: {
+        $value: '{base.color.scale.gray.2}',
+        alpha: 0,
+        $type: 'color'
+      }
+    },
+    small: {
+      $value: {
+        color: '{shadow.color.small}',
+        offsetX: '0px',
+        offsetY: '1px',
+        blur: '0px',
+        spread: "0px"
+      },
+      $type: 'shadow'
+    },
+    medium: {
+      $value: {
+        color: '{shadow.color.medium}',
+        offsetX: '0px',
+        offsetY: '3px',
+        blur: '6px',
+        spread: "0px"
+      },
+      $type: 'shadow'
+    },
+    large: {
+      $value: {
+        color: '{shadow.color.large}',
+        offsetX: '0px',
+        offsetY: '8px',
+        blur: '24px',
+        spread: "0px"
+      },
+      $type: 'shadow'
+    },
+    extraLarge: {
+      $value: {
+        color: '{shadow.color.extraLarge}',
+        offsetX: '0px',
+        offsetY: '12px',
+        blur: '28px',
+        spread: "0px"
+      },
+      $type: 'shadow'
+    },
+    // components
+    overlay: {
+      $value: [{
+        color: '{base.color.scale.gray.6}',
+        offsetX: '0px',
+        offsetY: '0px',
+        blur: '0px',
+        spread: "1px"
+      },{
+        color: '{shadow.color.medium}',
+        offsetX: '0px',
+        offsetY: '16px',
+        blur: '32px',
+        spread: "0px"
```

Comment:
is there a matcher on px here? Can you pass "0" with the same result?

Reply:
Currently there is no check for the values of the properties. I wasn't sure if this would be an overkill. Do you think we should validate that every property is of the correct type?

Reply:
Well, at the moment it IS valid. :D But of course it is not valid in w3c specs.